### ISSUE TITLE
Clarification on `ground_truth` usage during Solver training

### ISSUE THREAD

**Issue by DelicateNorman** [XMUDeepLIT/TTCS#13]
Hi authors,

First of all, thank you very much for releasing TTCS and providing such a clear implementation of the Synthesizer–Solver co-evolution framework. I found the idea of dynamically constructing capability-aligned curricula at test time very insightful, especially the use of self-consistency rewards to support label-free adaptation on challenging reasoning benchmarks.

I may have missed a preprocessing step, but I noticed that the Solver uses `reward_model["ground_truth"]` when available:

```python
if "ground_truth" in data[i].non_tensor_batch["reward_model"]:
    ground_truths.append(data[i].non_tensor_batch["reward_model"]["ground_truth"])
```

Meanwhile, original test examples appear to be copied directly into the Solver training data.

Could you please clarify whether `ground_truth` is removed from test examples beforehand? My understanding is that label-free test-time training should use majority-vote pseudo-labels for both original and synthetic questions.

Perhaps the code could explicitly restrict `ground_truth` usage to validation to avoid dataset-format-dependent behavior.

Thanks again for the excellent work!

**Comment by Acm1011** [comment]
Thanks for pointing this out. We would like to clarify that TTCS performs label-free test-time adaptation and does not use the ground-truth answers from test examples.

The ground_truth field in the code is only kept for compatibility with existing baseline pipelines and evaluation formats. In our TTCS pipeline, test examples do not contain this field, and the Solver uses majority-vote/self-consistency based pseudo-labels for both original and synthetic questions.
